src/team/team.controller.ts

```diff
@@ -111,22 +114,42 @@ export class TeamController {
 
             if (!teamDoc || !(teamDoc.adminIds).includes(req.person.genesisId)) {
                 log(LOG_LEVEL.INFO, parseLogData(TeamController.TEAM_MESSAGES.UNAUTHORIZED,
-                                                 'TeamController',
-                                                 '401',
-                                                 TeamController.TEAM_MESSAGES.NO_STACK));
+                    'TeamController',
+                    '401',
+                    TeamController.TEAM_MESSAGES.NO_STACK));
 
                 throw new Unauthorized(TeamController.TEAM_MESSAGES.UNAUTHORIZED);
             }
 
+            if (team.contactUserId) {
+                if (typeof (team.contactUserId) !== 'string' && team.contactUserId === "") {
```

Comment:
Need to check if the parameter given is not string **OR** empty string.
Both will never be true.
```suggestion
                if (typeof (team.contactUserId) !== 'string' || team.contactUserId === "") {
```